src/PiePay.sol

```diff
@@ -6,17 +6,25 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
 import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
 import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
-import "forge-std/console.sol";
+import "lib/forge-std/src/console.sol";
```

Comment:
I had to dick with this import to get it to work on my machine. Feel free to change it back if you need to